examples/miscellaneous/miscellaneous_ex12/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+#set -x
+
+source $LIBMESH_DIR/examples/run_common.sh
+
+example_name=miscellaneous_ex12
+
+options="-pc_type lu -pc_factor_mat_solver_package mumps"
```

Comment:
OK, good to know.  By the way, it seems like you are fairly experienced in using Eigen; I wonder if they have any direct solvers available that we might add interfaces for in the `EigenSparseLinearSolver`.  Not necessarily for this PR, more of a note to self for future work.